docs/pages/application-access/guides/connecting-apps.mdx

```diff
@@ -286,6 +288,10 @@ In the example above, `X-Internal-Trait` header will be populated with the value
 of internal user trait `logins` and `X-External-Trait` header will get the value
 of the user's external `env` trait coming from the identity provider.
 
+Additionally, the `{{internal.jwt}}` template variable will be replaced with
+the JWT token signed by Teleport that contains user identity information. See
```

Comment:
```suggestion
a JWT token signed by Teleport that contains user identity information. See
```
Since this is the first time we introduce this particular JWT.